freqtrade/exchange/binance.py

```diff
@@ -97,41 +102,114 @@ def additional_exchange_init(self) -> None:
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
-    async def _async_get_historic_ohlcv(
+    def get_historic_ohlcv(
```

Comment:
Is there a particular reason you didn't remain in the async context, but moved this "up" from that? 
in the end, we'll still need the (same) async context for both download methods - just that now we need "run_until_complete" to do actual work :D 

(for clarity: it's not necessarily a request for change - more an asking for reason)

Reply:
the async method returns an OHLCVRespnse which is kind of raw data, returning a dataframe is helpful for dev and makes the data flow concise.

Reply:
works for me ;) as said - just a confirming question